wagtailio/static/sass/components/_teaser.scss

```diff
@@ -0,0 +1,82 @@
+.teaser {
+    $root: &;
+    opacity: 1;
```

Comment:
Question: is there a reason to use opacity: 1 here?

Reply:
nope! this is my go to css rule to test that everything is working correctly and must have left it in by accident. Thanks!